src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -435,10 +435,26 @@ describe('<ComboBox/>', () => {
       );
       comboBox.find(TextField).simulate('click');
       dispatchKeyup(Key.DownArrow);
-      dispatchKeyup(Key.Enter);
```

Comment:
Should we add a regression test as well to assert the event doesn't propagate to the forms on submit?

Reply:
If its ok to have a test where the ComboBox is wrapped in `<form>` (since thats where the oSubmit call is made),  then yep i can add tests for those